app.json

```diff
@@ -43,6 +43,15 @@
             "useFrameworks": "static"
           }
         }
+      ],
+      [
```

Comment:
This values should be added to `.env`

Reply:
Additionally we could hide facebook button when developer will not pass this values, what do you think?